docs/Researcher/cli-reference/runai-port-forwarding.md

```diff
@@ -0,0 +1,70 @@
+# runai port-forward
+
+## Description
+
+Forward one or more local ports to the selected job. The forwarding session ends when the selected job terminates.
+
+Usage:
+
+To use port forwarding:
+
+1. Open a local terminal session.
+2. Find the target server using `<runai list -p <name of project>>`.
+3. Run the following command:
+     `runai port-forward <JOB_NAME> [flags]`
+
+!!!Note
+
+    1. You need to use port forwarding to get to the SSH on the server.
+    
+    2. You must use the SSH port that the server is listening to.
+ 
+    3. You must use a local session for port forwarding to the SSH port of the server. This is to ensure that you do not leave SSH sessions open for too long.
+
+    4. If a port forwarding session is terminated, all connections to that session will be terminated. You will need to rerun the `port-forward` command to reinstate port forwarding. Then, all previously connected users must reconnect.
+
+### Examples
+
+1. Port forward connections from localhost:8080 to <job-name> on port 8090.
+
+    `runai port-forward <job-name> --port 8080:8090 --address localhost`
```

Comment:
localhost is the default for --address, no need to specify this flag. I wouldn't have--address in the examples.

Reply:
We can state that, but what should appear there if you want to use something other than `localhost`?

Reply:
some ip address.. it's a very rare use case, in 99% of cases it will be localhost and so this flag should not be stated..